core/tutorials/buggyamb/Docs/README.md

```diff
@@ -0,0 +1,7 @@
+# Installation instructions
```

Comment:
In order for this sample to be discoverable we need to have the appropriate front matter. See https://review.docs.microsoft.com/en-us/help/contribute/samples/concepts/readme-template?branch=master

Also, I *think* this README should be at the root of the sample, so you'd have to move it down a directory - and update the refs to the other MD files. Here is a simple README with the minimal required front matter and content:

- https://github.com/dotnet/samples/blob/main/core/logging/custom-formatter/README.md
- https://docs.microsoft.com/en-us/samples/dotnet/samples/csharp-logging-fundamentals